README.md

```diff
@@ -46,79 +46,131 @@ pip install -r requirements.txt
+### MEMBRANE_SERVER_PUBLIC_KEY
 
-- **Description:** Path to the directory where client public keys are stored. These keys are used for JWT validation.
-- **Example:** `CLIENT_PUBLIC_KEYS_DIRECTORY=keys/public_keys`
+- **Description:** Path to the server's public key file used for verifying tokens.
+- **Example:** `MEMBRANE_SERVER_PUBLIC_KEY=keys/server_public.pem`
 
-### SERVER_PRIVATE_KEY
+### MEMBRANE_JWT_ACCESS_TOKEN_EXPIRE_SECONDS
 
-- **Description:** Path to the server's private key file. This key is used for creating and signing tokens.
-- **Example:** `SERVER_PRIVATE_KEY=keys/server_private.pem`
+- **Description:** Expiration time (in minutes) for the JWT access token.
+- **Example:** `MEMBRANE_JWT_ACCESS_TOKEN_EXPIRE_SECONDS=30`
 
-### SERVER_PUBLIC_KEY
+### MEMBRANE_JWT_EXPIRE_SECONDS
 
-- **Description:** Path to the server's public key file. This key is used for verifying tokens.
-- **Example:** `SERVER_PUBLIC_KEY=keys/server_public.pem`
+- **Description:** General JWT expiration time in minutes.
+- **Example:** `MEMBRANE_JWT_EXPIRE_SECONDS=60`
 
-### SESSION_LIFETIME_MINUTES
+### MEMBRANE_SESSION_LIFETIME_SECONDS
 
 - **Description:** Duration (in minutes) after which the session will expire.
-- **Example:** `SESSION_LIFETIME_MINUTES=30`
+- **Example:** `MEMBRANE_SESSION_LIFETIME_SECONDS=30`
 
-### REDIRECT_URL_TO_LOUIS_FRONTEND
+### MEMBRANE_SESSION_TYPE
 
-- **Description:** The URL to which the Louis login backend will redirect users, leading them to the Louis login frontend where they can provide an email address.
-- **Example:** `REDIRECT_URL_TO_LOUIS_FRONTEND=https://login.louisfrontend.com/`
+- **Description:** Specifies the storage for session data. Options: 'filesystem', 'redis', 'memcached', etc.
+- **Example:** `MEMBRANE_SESSION_TYPE=filesystem`
 
-Once you have defined all these variables, save and close the `.env` file. The Flask application will now use these environment variable values when it runs.
+### MEMBRANE_LOGGING_LEVEL
 
-### 7. Run the Main Flask Application:
+- **Description:** Specifies the logging level for the application.
+- **Example:** `MEMBRANE_LOGGING_LEVEL=INFO`
+
+### MEMBRANE_ALLOWED_EMAIL_DOMAINS
+
+- **Description:** List of email domains accepted by the application.
+- **Format:** Comma-separated list of domains.
+- **Example:** `MEMBRANE_ALLOWED_EMAIL_DOMAINS=gc.ca,canada.ca`
+
+### MEMBRANE_COMM_CONNECTION_STRING
+
+- **Description:** Connection string for the email service.
+- **Example:** `MEMBRANE_COMM_CONNECTION_STRING=smtp://example.com:587`
+
+### MEMBRANE_SENDER_EMAIL
+
+- **Description:** Email address that will send emails.
+- **Example:** `MEMBRANE_SENDER_EMAIL=noreply@example.com`
+
+### MEMBRANE_EMAIL_SUBJECT
+
+- **Description:** Subject line for outgoing emails.
+- **Example:** `MEMBRANE_EMAIL_SUBJECT=Your Subject Here`
+
+### MEMBRABE_EMAIL_SEND_SUCCESS
+
+- **Description:** Indicates if the email was sent successfully.
+- **Example:** `MEMBRABE_EMAIL_SEND_SUCCESS=true`
```

Comment:
Hmmm, isn't this variable? sometimes true, sometimes not? how is it a constant?

Reply:
also misspelled MEMBRANE

Reply:
done